Kernel/Graphics/Intel/DisplayConnectorGroup.cpp

```diff
@@ -293,7 +293,7 @@ u32 IntelDisplayConnectorGroup::read_from_general_register(RegisterOffset offset
     return value;
 }
 
-void IntelDisplayConnectorGroup::write_to_analog_output_register(AnalogOutputRegisterOffset index, u32 value) const
+void IntelDisplayConnectorGroup::write_to_analog_output_register(AnalogOutputRegisterOffset index, u32 value)
```

Comment:
The IntelDisplayConnectorGroup class is new to this PR, so please squash `Kernel/IntelGraphics: Mark write_to_analog_output_register as non-const` as well.